### PR TITLE
Adding interrupted status

### PR DESCRIPTION
**Pull request by timneil**
Added the interrupted status which is documented here - https://playwright.dev/docs/api/class-reporter#reporter-on-end
Introduced in 1.25.0 https://github.com/microsoft/playwright/releases/tag/v1.25.0

### CODE REVIEW

File: .idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Default ignored files
```
Comment (artemrudenko) [inline .idea/.gitignore:1]: Hi, i think that we need to exclude .idea files from package.

Comment (artemrudenko) [reply .idea/.gitignore:1]: These files are actual only while developing package and have no sence for ready one

Comment (timneil) [reply .idea/.gitignore:1]: Thanks for the response. I've removed the .idea files.

**Review by artemrudenko — APPROVED** [review]

File: .idea/.gitignore (hunk 1 above)
Comment (artemrudenko) [reply .idea/.gitignore]: Hi @timneil ,
Just figured out that you have updated package-lock.json with private npm repo links, and this breaks build. 
Could you please update lock file?
Best Regards,
Artem

**Review by artemrudenko — CHANGES_REQUESTED** [review]
Hi @timneil,

The build is still failing. Could you please confirm how you have updated the lock file? When I tried to install all the dependencies without your lock file, it created a new lock with paths that differ from yours. For example, instead of `"resolved": "https://registry.npmjs.org/repository/npm-group/@aashutoshrathi/word-wrap/-/word-wrap-1.2.6.tgz"`, I have `"resolved": "https://registry.npmjs.org/@aashutoshrathi/word-wrap/-/word-wrap-1.2.6.tgz"`. 

Could you please check once again and update the lock file?

Best regards,
Artem

**Comment by ranma42** [comment]
@artemrudenko merged my PR #13 (thank you! 🚀 ), which had some conflicting changes
If you need help rebasing this branch, ping me 😇

**Comment by artemrudenko** [comment]
> @artemrudenko merged my PR #13 (thank you! 🚀 ), which had some conflicting changes If you need help rebasing this branch, ping me 😇

Hi @ranma42, i have updated dependencies and performed small code refactoring, so it looks like now we have more conflicts. 

If you want i can simply copy and add your interrupted related code only and publish new version, or you can try to rebase your changes without adding package-lock and package.json changes. 
It's up to you.

Best Regards,
Artem

**Comment by artemrudenko** [comment]
Added by https://github.com/artemrudenko/playwright-teamcity-reporter/pull/17

**Pull request closed without merging**